File: visualBasic/IceBox/hello/HelloI.vb
' **********************************************************************
'
' Copyright (c) 2003-2017 ZeroC, Inc. All rights reserved.
'
' **********************************************************************

Imports Demo

Public Class HelloI
    Inherits HelloDisp_

    Public Overloads Overrides Sub sayHello(ByVal current As Ice.Current)
        System.Console.Out.WriteLine("Hello World!")
    End Sub
End Class

File: csharp/Ice/multicast/HelloI.cs
// Copyright (c) ZeroC, Inc. All rights reserved.

using System;
using System.Threading;
using Demo;

public class HelloI : HelloDisp_
{
    public override void
    sayHello(Ice.Current current, CancellationToken cancel)
    {
        System.Console.Out.WriteLine("Hello World!");
    }
}
